app/lib/lets-encrypt.server.ts

```diff
@@ -416,11 +416,19 @@ class LetsEncrypt {
     });
 
     this.#order = await this.#client.finalizeOrder(this.#order, csr);
-    const certificate = await this.#client.getCertificate(this.#order);
+
+    // fullChain is the publicCertificate + the intermediate certificate(s) for Let's Encrypt
+    const fullChain = await this.#client.getCertificate(this.#order);
+
+    // certificate is the public certificate, chain is the intermediate certificate(s) for Let's Encrypt
+    const [certificate, ...chainArray] = acme.crypto.splitPemChain(fullChain);
+    const chain = chainArray.join('');
 
     return {
       privateKey: key.toString(),
       certificate,
+      chain,
+      fullChain,
```

Comment:
I think the `fullChain` is redundant data. If I understand correctly, FE (or the certificate model) code would be able to re-join cert and chain at any time to get back the fullChain, very easily, and it is generally not advisable to store redundant data in the db.

Reply:
Good point.  Let's synthesize it in the front-end if needed, but not store in db.

Reply:
Alright, makes sense